src/applications/personalization/account/containers/AccountApp.jsx

```diff
@@ -11,41 +11,38 @@ import DowntimeNotification, {
 
 import { fetchMHVAccount } from 'platform/user/profile/actions';
 
-class AccountApp extends React.Component {
-  render() {
-    return (
-      <div>
-        <RequiredLoginView
-          serviceRequired={backendServices.USER_PROFILE}
-          user={this.props.user}
+function AccountApp(props) {
```

Comment:
I guess we don't have an ESLint rule to enforce `const AccountApp = ` over `function AccountApp` 😛 I'm used to seeing it as the former, but I see no difference in this case.